### PR TITLE
Update project-conf-files_services-rabbit.md

### PR DESCRIPTION
**Pull request by ryantfowler**
Added RabbitMq version in services declaration, as without this value the `ece-tools docker:build` command *will* fail.

## This PR is a:

- [ ] New topic
- [ ] Content update
- [ X ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will allow someone following the instructions from the documentation to actually do what they are intending to do, build a `docker-compose.yml` file without issues.

It does what I already typed above.

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services-rabbit.html

https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services-rabbit.html

Thank you for your contribution!

Before submitting this pull request, please make sure you have read our Contribution Guidelines and your PR meets our contribution standards:
https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md

Please fill out as much information as you can about your PR to help speed up the review process.
If your PR addresses an existing GitHub Issue, please refer to it in the title or Additional Information section to make the connection.

We may ask you for changes in your PR in order to meet the standards set in our Contribution Guidelines. PR's that do not comply with our guidelines may be closed at the maintainers' discretion.

Feel free to remove this section before creating this PR.
-->

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@ryantfowler thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/cloud/project/project-conf-files_services-rabbit.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ To enable RabbitMQ, add the following code with your installed version and alloc
 
 ```yaml
 rabbitmq:
-    type: rabbitmq
+    type: rabbitmq:3.5 ( or other supported version )
```
Comment (hguthrie) [inline guides/v2.1/cloud/project/project-conf-files_services-rabbit.md]:
```suggestion
    type: rabbitmq:<version>
```

Comment (ryantfowler) [reply guides/v2.1/cloud/project/project-conf-files_services-rabbit.md]: @hguthrie Change has been made.

**Comment by ryantfowler** [comment]
@hguthrie changes have been made. Please let me know. Thanks

**Comment by hguthrie** [comment]
running tests

**Pull request merged**